src/components/pages/career-details/career-details.tsx

```diff
@@ -16,6 +16,10 @@ interface CareerDetailsProps {
 
 const DEFAULT_CHANNEL_ID = '329206';
 
+const UTM_CHANNEL_MAP = new Map<string | null, string | null>([
```

Comment:
Looks overly complex. I would recommend to keep it a simple object:

```
const UTM_CHANNEL_MAP = {
  google_jobs_apply: '713571',
};
```

Also, add a comment above why we need to have this map.

Reply:
Making it less complex seems to cool, but if I make the map an Object I am not able to do this anymore:
![image](https://user-images.githubusercontent.com/78978542/160600623-9716430f-f70c-45f0-bc76-7c3528830714.png)

I liked that I dont have to think about the `utm_source` value and can just take it as key for my map to get the correct channel_id. (even if this whole thing makes only sense with more channels than only google jobs)

Is there a less complex way to do this or shall I just leave out the whole mapping thing and something like:

```
googleChannel = searchParams.get("utm_source") === "google_jobs_apply" ? GOOGLE_CHANNEL : undefinded;
```

Reply:
You can access an object with the array syntax:

```
const utmChannel = UTM_CHANNEL_MAP[searchParams.get('utm_source')]
```

Reply:
Thank you for the bracket notation tipp, I did not knew about this, and this way it feels convenient again 👍